apps/client/package.json

```diff
@@ -42,6 +41,7 @@
     "build:docker": "cross-env VITE_IS_DOCKER=true vite build",
     "build:localdocker": "cross-env NODE_ENV=local vite build",
     "lint": "eslint . --quiet",
+    "typegen": "chakra typegen src/theme/theme.ts",
```

Comment:
can you please add to DEVELOPMENT.md some documentation about this? we just need to know that chakra needs to run a type generation step and users may see type errors in the UI components if that does not happen

Reply:
Added in https://github.com/cpvalente/ontime/pull/1438/commits/25c675acc017901a14afd0a1689e3c4a320b31f5